pjlib/src/pj/ssl_sock_ossl.c

```diff
@@ -100,6 +100,17 @@
 
 #   include <openssl/obj_mac.h>
 
+#if OPENSSL_VERSION_NUMBER >= 0x10100000L || \
+    (defined(OPENSSL_API_COMPAT) && OPENSSL_API_COMPAT >= 0x10100000L)
+#  define X509_get_notBefore(x)     X509_getm_notBefore(x)
```

Comment:
There are similar definitions below, so perhaps combine them, to make sure there's no redefinition warning/error?